plugin/plugin-metadata.ts

```diff
@@ -0,0 +1,24 @@
+import { ConsolePluginBuildMetadata } from '@openshift-console/dynamic-plugin-sdk-webpack';
+
+const metadata: ConsolePluginBuildMetadata = {
+  name: 'ossmconsole',
+  version: '1.85.0',
```

Comment:
During our build process, we have update this version string. Because it is now in a new file, we should make sure our build process doesn't break. I'm not sure exactly where we do this - it is either in the github workflow or in some Makefile. I will look to try to find it.

Reply:
This looks like the place: https://github.com/kiali/openshift-servicemesh-plugin/blob/main/.github/workflows/release.yaml#L209

Reply:
Note that we still need to change "version:" here in package.json (there is one at the top of the file)... but now we need to do this version update in this plugihn-metadata.ts file, too.

Reply:
I think we want SOMETHING this:

```
        jq -r ".version |= \"${RELEASE_VERSION:1}\" plugin/package.json > plugin/package.json.tmp
        mv plugin/package.json.tmp plugin/package.json

        jq -r ".version |= \"${RELEASE_VERSION:1}\" plugin/plugin-metadata.json > plugin/plugin-metadata.json.tmp
        mv plugin/plugin-metadata.json.tmp plugin/plugin-metadata.json
```

But that probably isn't 100% correct because now the plugin version is inside a javascript file, not a json file so the "jq" tool likely will not work. So for that second set of commands we will need to use something like "sed" rather than "jq".

Reply:
see https://github.com/ferhoyos/openshift-servicemesh-plugin/pull/6